raiden-ts/src/utils/error.ts

```diff
@@ -0,0 +1,88 @@
+import * as t from 'io-ts';
+import { map } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { PfsErrorCodes } from '../path/errors';
+
+export const ErrorDetails = t.array(
```

Comment:
This codec declaration is equivalent to `{ key: string; value: string | number }[]`. If you wanna a mapping, you could use `t.record(t.union([t.string, t.number, ...]))`, which is equivalent to a proper indexed type, like `{ [k: string]: string | number | ... }`

Reply:
Ah was looking for that but took an easier right for now. Will change, thank you 🙏